### ISSUE TITLE
Icons in sub menus?

### ISSUE THREAD

**Issue by payrequestio** [livewire/flux#889]
### Flux version

not relevant

### Livewire version

not relevant

### What is the problem?

I guess it is not  possible to use icon in sub menus?
tried this but not works:    <flux:menu.submenu icon="language" heading="Language">

### Code snippets

`<flux:menu.submenu icon="language" heading="Language">`

### How do you expect it to work?

Make it possible to use icons in submenus

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by jeffchown** [comment]
@payrequestio That is correct - currently, `flux:menu.submenu` does not accept the `icon` attribute.

One workaround is e.g.:

```
<flux:menu.submenu>
    <x-slot:heading>
        <flux:icon.language class="mr-2 size-4"/>
        Language
    </x-slot:heading>

    ...flux:menu.item... 
    ...flux:menu.item... 
    ...flux:menu.item... 
</flux:menu.submenu>
```

**Comment by joshhanley** [comment]
@payrequestio thanks for reporting! I've submitted a PR to add icon support to submenus.
